src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum Color {

```

Comment:
Consider adding a method to get a random color from the enum. This would be useful for the ColorSupplier class to generate random colors.